moon/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-name: moon
-version: 1.0.7
 appVersion: 1.8.3
 description: Moon Helm chart
+name: moon
+version: 1.1.5
```

Comment:
This change makes is useless. Version is being set automatically when deploying chart: https://github.com/aerokube/charts/blob/master/ci/build.sh#L13.